.internal-ci/helm/consensus-node/templates/node-deployment.yaml

```
@@ -28,20 +28,28 @@ spec:
         client-load-balanced: 'true'
         {{- end }}
     spec:
-      nodeSelector:
-        {{- toYaml .Values.node.nodeSelector | nindent 8 }}
+      # Try to balance pods across zones
+      topologySpreadConstraints:
+      - topologyKey: topology.kubernetes.io/zone
+        maxSkew: 1
+        # Wait until we have nodes
+        whenUnsatisfiable: DoNotSchedule
+        labelSelector:
+          matchLabels:
+            # match app and helm chart version
+            app: consensus-node
+            helm.sh/chart: {{ include "consensusNode.chart" . }}
```

The additional labels are used in other configs but not this one.
```suggestion
            helm.sh/chart: {{ include "consensusNode.chart" . }}
            {{- include "consensusNode.selectorLabels" . | nindent 6 }}
```

---

This is intentional, consensus gets deployed in multiple helm "releases". So we want don't want to restrict matching on release name, which using selectorLabels would do.